.changelog/unreleased/Bug Fixes/14-blockNumber-in-log.md

```diff
@@ -0,0 +1 @@
+- fixes the `blockNumber` reported a debug log ([#14](https://github.com/noble-assets/jester/pull/14))
```

Comment:
verbiage is a bit weird here

Reply:
Thank you. Not sure where I was going with this.